website/source/guides/external/lxc.html.md

```diff
@@ -0,0 +1,208 @@
+---
+layout: "guides"
+page_title: "LXC"
+sidebar_current: "guides-external"
+description: |-
+  Guide for using LXC external task driver plugin.
+---
+
+## LXC
+
+The `lxc` driver provides an interface for using LXC for running application containers. You can download the external LXC driver [here][lxc_driver_download]. This guide is compatible with Nomad 0.9 and above. If you are using an older version of Nomad, see the [LXC][lxc-docs] driver documentation.
+
+## Reference Material
+
+- Official [LXC][linux-containers] documentation
+- Nomad [LXC][lxc-docs] external driver documentation
+
+## Estimated Time to Complete
+
+20 minutes
+
+## Challenge
+
+You need to deploy a workload using [Linux Containers][linux-containers-home]. Configure the client nodes that need to run this workload appropriately. You will also need to install the `lxc-templates` package which will provide the templates needed to start your containers.
+
+## Solution
+
+Install and configure the LXC external driver plugin. Verify the configuration on the client node and deploy the application.
+
+## Prerequisites
+
+To perform the tasks described in this guide, you need to have a Nomad
+environment with Consul installed. You can use this
+[repo](https://github.com/hashicorp/nomad/tree/master/terraform#provision-a-nomad-cluster-in-the-cloud) to easily provision a sandbox environment. This guide will assume a cluster with one server node and one client node.
+
+-> **Please Note:** This guide is for demo purposes and is only using a single server node. In a production cluster, 3 or 5 server nodes are recommended.
+
+## Steps
+
+### Step 1: Verify Client Node Configuration
+
+External drivers must be placed in the [plugin_dir][plugin_dir] directory which defaults to [`data_dir`][data_dir]`/plugins`. Verify the `data_dir` directory on the client node configuration. If you are using the environment provided by this guide, the client configuration is located at `/etc/nomad.d/nomad.hcl`. The configuration file will show you that the `data_dir` directory is `/opt/nomad/data`. The relevant snippet of the configuration file is shown below:
+
+```shell
+$ cat /etc/nomad.d/nomad.hcl 
+data_dir = "/opt/nomad/data"
+bind_addr = "0.0.0.0"
+...
+```
+
+### Step 2: Download and Install the LXC Driver 
+
+Make a directory called `plugins` in [plugin_dir][plugin_dir] (which is `/opt/nomad/data` in our case) and download/place the [LXC driver][lxc_driver_download] in it. The following sequences of commands illustrate this process:
+
+```shell
+$ sudo mkdir /opt/nomad/data/plugins
+$ curl -O /link/to/driver.zip
```

Comment:
The link should be https://releases.hashicorp.com/nomad-driver-lxc/0.1.0/nomad-driver-lxc_0.1.0_linux_amd64.zip I believe.  Will confirm once it's out.